### PR TITLE
On branch edburns-msft-jea-102-optional 

### PR DESCRIPTION
**Pull request by edburns**
https://dev.azure.com/jakarta-ee-azdo/jakarta-ee-azdo/_workitems/edit/104

modified:   specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

### CODE REVIEW

**Comment by hantsy** [comment]
Add a change log to describe the remove work.

File: specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc [hunk 1]
```diff
@@ -335,8 +335,8 @@ If a container supports Java SE 11 or a newer version of the Java SE platform, t
 
 During the development cycle for the current version of the Jakarta EE specification, the platform project considered several component specifications for inclusion in the platform. A consensus could not be reached on including these specifications in the platform. These specifications are considered as prospects for inclusion in a future version of the Platform specification.
 
-* [Jakarta MVC](https://jakarta.ee/specifications/mvc/)
-* [Jakarta NoSQL](https://jakarta.ee/specifications/nosql/)
+* https://jakarta.ee/specifications/mvc/[Jakarta MVC]
+* https://jakarta.ee/specifications/nosql/[Jakarta NoSQL]
```
Comment (ivargrimstad) [inline specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc:339]: NoSQL has not released a 1.0 yet. Isn't that one of the criteria for a prospective specification?

**Review by ivargrimstad — COMMENTED** [review]
Just the comment for NoSQL. Otherwise LGTM

**Review by ivargrimstad — APPROVED** [review]

File: specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc [hunk 2]
```diff
@@ -335,8 +335,7 @@ If a container supports Java SE 11 or a newer version of the Java SE platform, t
 
 During the development cycle for the current version of the Jakarta EE specification, the platform project considered several component specifications for inclusion in the platform. A consensus could not be reached on including these specifications in the platform. These specifications are considered as prospects for inclusion in a future version of the Platform specification.
 
-* [Jakarta MVC](https://jakarta.ee/specifications/mvc/)
-* [Jakarta NoSQL](https://jakarta.ee/specifications/nosql/)
```
Comment (Emily-Jiang) [inline specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc:339]: Why was this line removed?

Comment (ivargrimstad) [reply specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc:339]: One of the requirements for being listed as a Prospective spec is to have released at least one final version of the specification. NoSQL is still a beta.
The can be added back if they release a final version before EE 11.
